test/test-framework/pkg/apis/cache/v1alpha1/dummy_types.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type NoKindSpec struct {
+	// Not included in anything, no kind type
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
+	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors=true
+	Size int32 `json:"size"`
```

Comment:
Instead of creating a new kind in the mock data, wdyt about add the annotations in the kinds which already exists?

Reply:
These types allow a more complex set of test cases than the memcached-operator would allow. Rather than creating a whole new directory of test data I opted to add one file.

Reply:
Could we add descriptors in the others as well?

Reply:
Why does a new CSV need to be generated? There's no CRD for the dummy types so they will not be included in the CSV.

Reply:
Because the goal of this PR is gen the CSV file with the annotations added on this example. 
Also, IMHO we should keep the test-framework always updated since it is used by the tests and then we can properly validate the changes. Is it makes sense?

Reply:
The goal of this PR is to do that for operator projects, not necessarily our test data. The dummy types do not have CRD's, therefore will not be included in the resulting CSV. The dummy type's sole purpose is to act as test data, much like the rest of `test/test-framework`, for testing descriptor parsing code. We can discuss moving these types elsewhere, but they will not be a part of any CSV.